generate.pl

```diff
@@ -56,12 +56,14 @@ sub die_with_sample {
        patch \
        # procps \
        # zlib1g-dev \
-       xz-utils
+       xz-utils \
+       lib32z1-dev \
```

Comment:
Late notice, but this will break arm64 builds as there's no `lib32z1-dev` there - we should be pulling `zlib1g-dev`.

Will include fix in #119

Reply:
Thanks for fixing it!